julia.html.markdown

```diff
@@ -78,7 +78,7 @@ false
 1 < 2 < 3 # => true
 2 < 3 < 2 # => false
 
-# Strings are created with "
+# AbstractStrings are created with "
```

Comment:
Here's the link to the rationale

https://github.com/JuliaLang/julia/pull/8872